ansible/roles/etcd/tasks/iptables.yml

```diff
@@ -18,10 +18,9 @@
 
 - name: Open etcd client legacy port with iptables
   command: /sbin/iptables -I INPUT 1 -p tcp --dport {{ item }} -j ACCEPT -m comment --comment "etcd-legacy"
-  when: "'etcd-legacy' not in iptablesrules.stdout and networking == contiv"
+  when: "'etcd-legacy' not in iptablesrules.stdout and networking == 'contiv'"
```

Comment:
this should be the fix to the problem.

Do you mind updating ansible//roles/etcd/tasks/firewalld.yml to quote contiv in:

when: networking == contiv

and test in Atomic? I would like to make sure the firewall/iptables implementation are as similar as possible.